mep/books/models.py

```diff
@@ -288,6 +288,11 @@ def creator_by_type(self, creator_type):
         return [creator.person for creator in self.creator_set.all()
                 if creator.creator_type.name == creator_type]
 
+    def get_absolute_url(self):
+        '''Return the public url to view book's detail page'''
+        # NOTE using pk as slug for now
+        return reverse('books:book-detail', args=[self.pk])
```

Comment:
Oh! I should add updating this to use slugs to the issue where I'm implementing book slugs!

Reply:
I figured there would be some of that! there are also a bunch of view tests that use `reverse()` that will need updating, but you probably saw that too.